### PR TITLE
Website simplification and Coq platform update.

### PR DESCRIPTION
**Pull request by Zimmi48**
This is a major update to the website home page and Coq installation pages to put the focus on the Coq platform installation method (cc @gares @MSoegtropIMC). It also reduces the importance of the current version number on the website in relation with [CEP#52](https://github.com/coq/ceps/pull/52) (but we still need it, mostly in the opam using page). For the homepage, I also tried to think at what we would like to put forward if the design of the website was reworked, but without doing any change to the design yet.

The diff for the index page and the download page might be a bit hard to follow, so here are some preview screenshots:

2021-01-13-195725_613x904_scrot
2021-01-13-195647_617x797_scrot

cc @jfehrle (I cannot request a review from you on this repository)

### CODE REVIEW

**Comment by Zimmi48** [comment]
Note that for now, "Binary installers (Windows and macOS)" still link to the Coq GitHub release page, but we may need to update this to point to the Coq platform repository instead in the future.

**Comment by Zimmi48** [comment]
Another note is that we should get major user interfaces compatible with the Snap package (cf. https://github.com/coq-community/vscoq/issues/192 for VsCoq) before actively recommending the Snap package like this update does.

**Comment by Zimmi48** [comment]
Also cc @CohenCyril: I point to Nix as an alternative advance method and link to the nixpkgs-unstable manual.

File: pages/about-coq.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
 <div class="frameworklabel">Credits</div>
 <div class="frameworkcontent">
 
-    <p>Coq is the result of about 30 years of research. It started in
+    <p>Coq is the result of more than 30 years of research. It started in
     1984 from an implementation of the Calculus of Constructions at
     INRIA-Rocquencourt by Thierry Coquand and Gérard Huet. In 1991,
```
Comment (jfehrle) [inline pages/about-coq.html:68]:
```suggestion
    Inria-Rocquencourt by Thierry Coquand and Gérard Huet. In 1991,
```

File: pages/about-coq.html [hunk 2]
```diff
@@ -63,7 +63,7 @@
...
     Christine Paulin extended it to the Calculus of Inductive
```
Comment (jfehrle) [inline pages/about-coq.html:69]: (below):

"More than a hundred people have contributed"

That sounds low, maybe.

File: pages/about-coq.html [hunk 3]
```diff
@@ -78,9 +78,9 @@
     synthetic <a href="/who-did-what-in-coq">who did what</a>
     table).</p>
 
-    <p>The development is coordinated by the ADT Coq (Action for Technological Development), that gathers the teams involved in the implementation of the Coq Proof Assistant. The teams registered in the ADT are the <a href="http://www.inria.fr">INRIA</a> projects <a href="http://www.pps.univ-paris-diderot.fr/pi.r2/">πr²</a> and <a href="http://www-sop.inria.fr/marelle/">Marelle</a>, and the team <a href="http://cedric.cnam.fr/AfficheEquipe.php?id=7&amp;lang=fr">CPR</a> from <a href="http://www.cnam.fr">CNAM</a>.
-</p>
-
+    <p>The development is currently coordinated by a core development
+    team made of Inria researchers and engineers in various
+    project-teams in France.</p>
```
Comment (jfehrle) [inline pages/about-coq.html:83]:
```suggestion
    <p>Development is now coordinated by a core development
    team of Inria researchers and engineers in various
    project teams in France.</p>
```

File: pages/about-coq.html [hunk 4]
```diff
@@ -78,9 +78,9 @@
...
 
     <p> Coq is written in the <a href="https://ocaml.org/index.fr.html">OCaml</a> language, with a bit of C. It is
```
Comment (jfehrle) [inline pages/about-coq.html:85]:
```suggestion
    <p> Coq is written in the <a href="https://ocaml.org/index.fr.html">OCaml</a> language, with a small amount of C. It is
```

File: pages/documentation.html [hunk 5]
```diff
@@ -12,7 +12,10 @@
 <ul>
 
 <li>the <a href="/distrib/current/refman/">Reference Manual</a>, which
-is the complete, authoritative source of documentation for Coq;</li>
+is the complete, authoritative source of documentation for Coq; it
+contains
+the <a href="/distrib/current/refman/changes.html">changelog</a>,
+which we recommend you read when you upgrade your version of Coq;</li>
```
Comment (jfehrle) [inline pages/documentation.html:18]:
```suggestion
is the authoritative source of documentation for Coq.  It
contains
a <a href="/distrib/current/refman/changes.html">changelog</a>
describing updates to Coq,
which we recommend you read when you upgrade Coq;</li>
```

I hesitate to say the doc is complete.

File: pages/download.html [hunk 6]
```diff
@@ -1,54 +1,63 @@
...
 
-<p>For downloads (in particular to get installers for Windows and Mac OS, the PDF
-  manual, or a tarball of the sources), please go to
-  <a href="<#CURRENTVERSIONURL>">the release page on GitHub</a>.</p>
+<p>Coq has a rich ecosystem of external packages (libraries and
+plugins) that extend it and make it more powerful.  The Coq platform
+provides an easy way to install Coq and a consistent set of packages
+on Windows, macOS and many Linux distributions.</p>
+
+<p>Beginners are recommended to use one of the binary installers: we
```
Comment (jfehrle) [inline pages/download.html:15]:
```suggestion
<p>Beginners are encouraged to use one of the binary installers: we
```

Or, perhaps better, "We encourage ..."

File: pages/download.html [hunk 7]
```diff
@@ -1,54 +1,63 @@
...
+provide <a href="https://github.com/coq/coq/releases/latest">binary
+installers</a> for Windows and macOS as well as
+a <a href="https://snapcraft.io/coq-prover">Snap package</a>
+compatible with many Linux distributions.</p>
+
+<p>Experienced users are advised to run the scripts provided
+by <a href="https://github.com/coq/platform">the platform</a> to
+install from sources as this will allow them to further extend the
+resulting package set <a href="/opam-using.html">using opam</a>.</p>
```
Comment (jfehrle) [inline pages/download.html:24]:
```suggestion
<p>Experienced users are advised to run the scripts provided
by <a href="https://github.com/coq/platform">the "Coq platform"</a> to
install from sources as this will allow them to further extend the
resulting package set <a href="/opam-using.html">using opam</a>.</p>
```

Seems like building Coq from sources and building with coq/platform are separate options that should be described separately.  This confused me for a moment because "Coq platform" is a very generic term that you're using here as a defined term.

File: pages/download.html [hunk 8]
```diff
@@ -1,54 +1,63 @@
...
-<p>
-The Coq wiki contains three helpful pages detailing various ways of
-installing Coq for the three most standard operating systems:
-</p>
+<p>Coq is available
+through <a href="https://repology.org/metapackage/coq/versions">many
+package managers</a>, including most Linux distribution package
+managers.  However, in most cases, the available version will not be
+the latest version and, more importantly, installing Coq this way
+doesn't allow to extend it with external packages, so this
```
Comment (jfehrle) [inline pages/download.html:47]:
```suggestion
doesn't allow to extending it with external packages, so this
```

File: pages/download.html [hunk 9]
```diff
@@ -1,54 +1,63 @@
...
 <div class="frameworkcontent">
 
-<p>
-The Coq wiki contains three helpful pages detailing various ways of
-installing Coq for the three most standard operating systems:
-</p>
+<p>Coq is available
+through <a href="https://repology.org/metapackage/coq/versions">many
+package managers</a>, including most Linux distribution package
+managers.  However, in most cases, the available version will not be
```
Comment (jfehrle) [inline pages/download.html:45]:
```suggestion
managers.  However, in most cases, the available version is not
```

File: pages/download.html [hunk 10]
```diff
@@ -57,20 +66,24 @@
...
-and downloadable from our Git repository using command:
-<tt>git clone https://github.com/coq/coq.git</tt>.
+<p>The development version of Coq and its development history can be
+browsed on its <a href="https://github.com/coq/coq">GitHub
+repository</a>.
 </p>
 
-<p>Most of the previous versions of Coq are available <a href="<#RELEASES>">on
-GitHub</a> or <a href="/distrib/">here</a>.</p>
+<p>When upgrading your version of Coq, make sure you read
```
Comment (jfehrle) [inline pages/download.html:74]:
```suggestion
<p>When you upgrade your version of Coq, be sure to read
```

File: pages/opam-using.html [hunk 11]
```diff
@@ -19,6 +19,21 @@ <h2>What is opam?</h2>
...
+provide interactive scripts that allow installing Coq and a standard
+set of packages through opam without having to learn anything about
+opam.  We recommend that you use these scripts.  If you do, you can
+skip directly to <a href="#coq-packages">Using opam to install Coq
+packages</a> to learn how to extend the package set that you got.</p>
+
+<p>Note that the platform scripts are compatible with an existing opam
+installation and will create a fresh <a href="#switch">switch.</a></p>
+
+<p>If you prefer doing a fully manual installation, you can proceed to
```
Comment (jfehrle) [inline pages/opam-using.html:34]:
```suggestion
<p>If you prefer to do a fully manual installation, you can proceed to
```

File: pages/opam-using.html [hunk 12]
```diff
@@ -19,6 +19,21 @@ <h2>What is opam?</h2>
 <p>Follow the instructions below to install the last stable version of Coq and
 additional packages. The instructions target an opam newcomer.</p>
 
+<h2 id="platform">The Coq platform scripts</h2>
+
+<p>The <a href="https://github.com/coq/platform">Coq platform</a>
+provide interactive scripts that allow installing Coq and a standard
```
Comment (jfehrle) [inline pages/opam-using.html:25]:
```suggestion
provides interactive scripts that allow installing Coq and a standard
```

File: pages/opam-using.html [hunk 13]
```diff
@@ -19,6 +19,21 @@ <h2>What is opam?</h2>
...
+set of packages through opam without having to learn anything about
+opam.  We recommend that you use these scripts.  If you do, you can
+skip directly to <a href="#coq-packages">Using opam to install Coq
+packages</a> to learn how to extend the package set that you got.</p>
```
Comment (jfehrle) [inline pages/opam-using.html:29]: "extend" meaning "add additional packages" rather than "edit downloaded packages"?  If so, I'd replace "extend" with "add additional packages".

File: pages/opam-using.html [hunk 14]
```diff
@@ -19,6 +19,21 @@ <h2>What is opam?</h2>
...
+
+<p>Note that the platform scripts are compatible with an existing opam
+installation and will create a fresh <a href="#switch">switch.</a></p>
```
Comment (jfehrle) [inline pages/opam-using.html:32]:
```suggestion
<p>Note that the platform scripts are compatible with existing opam
installations.. They will create a fresh <a href="#switch">switch.</a></p>
```

**Review by jfehrle — COMMENTED** [review]
Looks better.  A variety of comments.  Not sure I love the page layout though.

Would be fun to add a photo to add interest.  How about this one (a free image from https://www.pexels.com/search/chicken/)?

image

Or this one for the Coq Community?

image

File: pages/about-coq.html (hunk 1 above)
Comment (gares) [reply pages/about-coq.html:68]: Back then, it used to be all caps :-/

File: pages/about-coq.html (hunk 3 above)
Comment (gares) [reply pages/about-coq.html:83]: I believe "project-team" is a french thing. I'd zap "in various ... in France.".

File: pages/about-coq.html (hunk 4 above)
Comment (gares) [reply pages/about-coq.html:85]: I'm not even sure this detail is interesting, and for sure it is not a distinguished feature (on the contrary OCaml has his place here).

**Comment by gares** [comment]
Many thanks for the update!

**Comment by gares** [comment]
>  Not sure I love the page layout though.

A "refresh" of the webpage style was planned by @maximedenes (to be outsourced to professionals).
While I could not agree more with you Jim on this point, I think this PR should focus on the contents.

File: pages/about-coq.html (hunk 3 above)
Comment (MSoegtropIMC) [reply pages/about-coq.html:83]: I would leave away any temporal reference like "currently" or "now".

File: pages/download.html (hunk 6 above)
Comment (MSoegtropIMC) [reply pages/download.html:15]: One might add some reasoning here:
- Binary installers = fast (a few minutes) but less flexible (cannot easily add additional packages)
- Build from sources = slow (1 ... 5 hours) but very flexible

**Comment by gares** [comment]
@Zimmi48 FYI the snap package page has a button (or maybe I'm the only one that sees it being admin) to generate html code for pointing to the package (a badge).

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : one thought: should we put this online before a Coq Platform 8.13.0.0 is published? I had a discussion on this with @gares. Essentially the options are:

- Publish Coq Platform 8.13.0.0 now with the traditional feature set (this is tested to work) and publish 8.13.0.1 with extended feature set in one month.
- Publish Coq Platform 8.13.0.0 with an extended feature set in one month.

Since we are in a transitional situation I would tend to the first option, but @gares preferred the second. I think we both don't have very strong opinions on this, but this should be coordinated with this PR.

**Comment by Zimmi48** [comment]
> Maybe I'm the only one that sees it being admin, in that case I can give you the pwd to look at the preview.

I noticed it but it looked too heavy for this page.

**Comment by Zimmi48** [comment]
> Since we are in a transitional situation I would tend to the first option, but @gares preferred the second. I think we both don't have very strong opinions on this, but this should be coordinated with this PR.

I would also prefer the first option.

**Comment by gares** [comment]
One thing I don't like much is to have two platforms for the same coq release with different feature sets. The 4th digit becomes very important then, to answer the question "is package X in the platform for 8.13". This is probably why I preferred option 2, but it goes in hand with the meaning of that digit. And for the records, I'm also a bit skeptical about inheriting the third digit from Coq (what 8.13.1.0 means? does it have the same features of 8.13.0.1 or 8.13.0.0?)

**Comment by gares** [comment]
> . I think we both don't have very strong opinions on this, but this should be coordinated with this PR.

+1

**Comment by MSoegtropIMC** [comment]
> One thing I don't like much is to have two platforms for the same coq release with different feature sets.

The idea is that the content only ever increases, so that 8.13.1.0 will have the same or larger content than 8.13.0.1. We just need some efficient way to add packages without delaying this too long and it is not unlikely that the Coq platform will need more maintenance releases than Coq itself - that's the reason for the 4th digit.

What I plan is to have one more level (the first question the script asks, coq only, ide or full) called "extended", but I didn't plan as yet what goes in what level and if "full" should be renamed to "standard" or "default".

**Comment by Zimmi48** [comment]
Yes, the standard vs extended set is a good idea. I agree with the versioning scheme concern, but we should probably move this discussion somewhere else -> coq/platform#22.

File: pages/about-coq.html (hunk 3 above)
Comment (jfehrle) [reply pages/about-coq.html:83]: Also, is it appropriate to define the core team as Inria personnel only?  Could we be inclusive?  Perhaps just say "a core development team, many of whom are Inria researchers and engineers".  Some developers who are very active are not in France.

**Comment by jfehrle** [comment]
FWIW, one way to facilitate reviewing website updates is to temporarily make the new material available on a non-standard TCP port or under a different DNS name, which reviewers can include in their URL while others won't find it.  This may be easy or way too much work depending on the web site technology.

**Comment by Zimmi48** [comment]
> FWIW, one way to facilitate reviewing website updates is to temporarily make the new material available on a non-standard TCP port or under a different DNS name, which reviewers can include in their URL while others won't find it. This may be easy or way too much work depending on the web site technology.

Unfortunately, the current infrastructure is a big mess. So this is not an option until it is replaced by something simpler. But reviewers can also check out the branch and run `make && make run` to get a preview, with just a few links broken (only OCaml and Python are needed).

**Comment by Zimmi48** [comment]
> Would be fun to add a photo to add interest. How about this one (a free image from https://www.pexels.com/search/chicken/)?

I like this idea but consider it part of design, so to keep for later.

File: pages/about-coq.html (hunk 4 above)
Comment (Zimmi48) [reply pages/about-coq.html:85]: So just remove the mention of C?

File: pages/download.html (hunk 7 above)
Comment (Zimmi48) [reply pages/download.html:24]: OK to add the Coq qualifier, but why the quotes (especially in a paragraph about the Coq platform)?

File: pages/index.html [hunk 15]
```diff
@@ -17,112 +17,87 @@
...
   and <a href="/cocorico/CoqInTheClassroom">teaching</a>.
   </p>
+
+  <p>The reference documentation for Coq are the <a
+  href="/distrib/current/refman/"> Reference Manual </a> and the
+  documentation of the <a href="/distrib/current/stdlib/">
+  Standard Library</a>.
+  However, to learn about Coq, it's best to start with a tutorial
+  or book. Go to our <a href="/documentation">documentation page</a>
+  for a list of such tutorials and books.</p>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html]: Any better suggestion?

File: pages/index.html [hunk 16]
```diff
@@ -17,112 +17,87 @@
...
+<li><a href="/distrib/current/refman/"> Reference Manual </a></li>
+<li><a href="/distrib/current/stdlib/"> Standard Library </a></li>
 </ul>
 </div>
 </div>
 
-
 <div class="framework">
-<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/download">How to get it?</a></div>
+<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/download">Try / Install Coq</a></div>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html:41]: Yes, thanks. I was not satisfied by this title. Maybe just "Run Coq" though, to be consistent with the rest of the titles.

File: pages/index.html [hunk 17]
```diff
@@ -17,112 +17,87 @@
...
-to try Coq without installing anything, we recommend you
-use <a href="https://jscoq.github.io">jsCoq</a>.
-</p>
+<p>You don't need to install Coq to get started: run Coq in your browser using
+<a href="https://jscoq.github.io">jsCoq</a>!</p>
+
+<p>To go beyond, you can proceed to <a href="/download">install the
+Coq platform</a> on your machine and
+a <a href="user-interfaces.html">user interface</a> of your
+choice.</p>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html]: >I expect most people would eventually install Coq on their own system. Is that right? Can we give a sentence saying why users will want to install on their system--what added features, etc.?

That's right. Getting Coq installed gives you a proper way to conduct projects with multiple files, gives you access to faster reduction machines, etc.

>BTW, I find it very hard to pick out the links in the text. Can you change the highlight color for all the links to something more vivid?

Again a design issue. I'd rather not touch that in this PR, especially as @maximedenes is planning to prepare a refresh of the design website.

File: pages/index.html [hunk 18]
```diff
@@ -17,112 +17,87 @@
...
-</ul>
-</div>
-</div>
+<p>Coq has a strong and active user community. They are contributing
+formal developments, extensions of Coq
+(see <a href="/packages.html">Coq Package Index</a>), and tools based
+on Coq (see <a href="/related-tools.html">Related Tools</a>).  We have
+a (multi-lingual) <a href="https://coq.discourse.group">Discourse
+forum</a> and a <a href="https://coq.zulipchat.com">Zulip chat</a>.
+</p>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html]: >"formal developments" - you mean proofs, definitions, ?? The term doesn't do much for me.

It's not my words, but yes formal definitions and proofs.

>"tools based on Coq" - includes both tools built on top of Coq and tools for use with Coq IIUC, so that name isn't a precise fit.

Right. Any better suggestion?

>Is it worth suggesting how to reach individuals? Or seeing a list of the main contributors? OTOH some prefer to be less visible.

It would indeed be a good thing to get a team page. This is something that has been wanted for a long time. But much beyond the scope of this PR.

>Also, BTW, the Related Tools page has a lot of bullet points that start with "XXX is a/an YYYY". Would be less repetitive to change these to "XXX - a YYY"

Also, this page would be worth keeping more up-to-date. I'm thinking maybe a manually curated page is not a good fit for this.

File: pages/index.html [hunk 19]
```diff
@@ -17,112 +17,87 @@
...
-(see <a href="/related-tools.html">Related Tools</a>).
-We have a (multi-lingual) <a href="https://coq.discourse.group">Discourse forum</a>
-and a <a href="https://coq.zulipchat.com">Zulip chat</a>.
-</p>
-
+    <p>You can contribute to the development of Coq by reporting bugs,
+    submitting pull requests, improving the documentation, and in many other
+    ways. We are also looking for sponsors who want to contribute financially
+    through the <a href="/consortium">Coq Consortium</a>. If you are interested,
+    please get in touch!</p>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html]: I'm not sure what the current status is. There might have been a US foundation involved in the process at some point (since most initial contributors were from the US) but I'm not sure it's still the case today.

File: pages/opam-using.html (hunk 13 above)
Comment (Zimmi48) [reply pages/opam-using.html:29]: You can do both, but I indeed mean the former.

**Comment by jfehrle** [comment]
I noticed 2 other typos in documentation.html:

ressources
targetted

File: pages/index.html (hunk 16 above)
[the remark this answers is not in the document]
Comment (jfehrle) [reply pages/index.html:41]: Actually I think the others would read a little better as "Learning ..." and "Contributing ...".  "Contribute to Coq" reads like a command (Contribute is a verb) whereas in "Contributing to Coq", "Contributing" is, I think a noun (ie the gerund of the verb).  OTOH, FWIW, the dictionary lists "learning" as a noun but not "contributing".

File: pages/index.html (hunk 15 above)
[the remark this answers is not in the document]
Comment (jfehrle) [reply pages/index.html]: "Resources" (a bit vague; libraries could be considered resources)?
How about "Learning more about Coq" or "Resources for Learning about Coq"?

I guess "Miscellaneous" and "Stuff" would not cut it :-).

File: pages/index.html (hunk 17 above)
[the remark this answers is not in the document]
Comment (jfehrle) [reply pages/index.html]: If you're willing to ask Maxime and he's O with it, you could increase the red level.  Should be one line in a CSS file.  It would affect all pages in the website.  The links are barely discernable--it's an easy way to make it a lot more usable.

If I may offer an opinion, it would nice if the developers were invited to offer suggestions on the website (but probably leaving the decisions to Maxime).  Also incremental improvement and review is a superior to the "waterfall" development model (both for websites and software).

File: pages/index.html (hunk 18 above)
[the remark this answers is not in the document]
Comment (jfehrle) [reply pages/index.html]: How about "tools for use with Coq and based on Coq"?

File: pages/index.html (hunk 19 above)
[the remark this answers is not in the document]
Comment (jfehrle) [reply pages/index.html]: For consortium members such as members of the DeepSpec project, their payments would be considered business expenses--fully deductible without a foundation.  The non profit would benefit individuals making charitable contributions who aren't getting a personal benefit it return.

File: pages/about-coq.html (hunk 1 above)
Comment (jfehrle) [reply pages/about-coq.html:68]: It could be a girl's name...

File: pages/about-coq.html (hunk 2 above)
Comment (jfehrle) [reply pages/about-coq.html:69]: Sure.  FWIW, saying "hundreds of people" would be technically correct but a little misleading.

File: pages/about-coq.html (hunk 3 above)
Comment (jfehrle) [reply pages/about-coq.html:83]: What distinguishes core team members from other developers?  Is there a secret handshake?

File: pages/about-coq.html (hunk 4 above)
Comment (jfehrle) [reply pages/about-coq.html:85]: Removing "C" is fine by me.

File: pages/download.html (hunk 7 above)
Comment (jfehrle) [reply pages/download.html:24]: Because "the Java platform", "the Python platform", etc. is generic phrasing.  If I build Coq from sources, I think I've built the Coq platform, whereas here IIUC it's being used as a defined term  (hence the quotes) to refer to what you get from building from sources _plus_ a bunch of additional libraries.  Might help to make this distinction more explicit?  Does that make any sense?

File: pages/opam-using.html (hunk 13 above)
Comment (jfehrle) [reply pages/opam-using.html:29]: Maybe say the former then.  I tend to think that 90% of users will not extend packages.

File: pages/download.html [hunk 20]
```diff
@@ -1,54 +1,63 @@
...
-<p>
-The Coq wiki contains three helpful pages detailing various ways of
-installing Coq for the three most standard operating systems:
-</p>
+<p>Coq is available
+through <a href="https://repology.org/metapackage/coq/versions">many
+package managers</a>, including most Linux distribution package
+managers.  However, in most cases, the available version is not
+the latest version and, more importantly, installing Coq this way
+doesn't allow extending it with external packages, so this
```
Comment (silene) [inline pages/download.html:47]: Fortunately, that is not true. Users did not wait for Opam and Nix before they started to use external packages with Coq. First, the good old way (`make && sudo make install`) still works for a large range of libraries. Second, these external packages might just be part of the Linux distribution too. For example, Fedora users can simply type `dnf install flocq`, since Flocq is part of the main repository. Thus, I would rather write:

"More importantly, these distributions might provide only a fraction of all the external packages available for Coq, thus requiring some manual compilation and installation to extend it."

File: pages/about-coq.html (hunk 3 above)
Comment (Zimmi48) [reply pages/about-coq.html:83]: I'll remove the paragraph entirely. It was an attempt to update this completely outdated paragraph, but the new one actually doesn't bring anything useful.

File: pages/download.html (hunk 7 above)
Comment (Zimmi48) [reply pages/download.html:24]: I think making "Coq platform" a link will do just as well as adding quotes.

File: pages/index.html [hunk 21]
```diff
@@ -4,7 +4,7 @@
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/about-coq">What is Coq?</a></div>
+<div class="frameworklabel"><a  style="color:black;font-weight:bold" href="/documentation">Learn about Coq</a></div>
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply pages/index.html:7]: Left as a to-do for later.

**Comment by Zimmi48** [comment]
@gares PR updated.

Two remarks:
- the links to the Coq platform are to https://github.com/coq/platform, but it could also be to https://github.com/coq/platform/releases
- the links to the Windows & macOS binary installers still point to https://github.com/coq/coq/releases/latest since this is anyway the only place where to find a macOS installer and the Windows installer should be mostly identical to the platform ones. The idea is to update this once all the installers can be found on the releases page of the platform.

**Comment by Zimmi48** [comment]
Following @MSoegtropIMC remark, I will update the PR to point to https://github.com/coq/platform/releases rather than https://github.com/coq/platform.

**Comment by Zimmi48** [comment]
Done.
PS: @MSoegtropIMC, I think the platform scripts should be advertised more clearly from the releases page. The natural way to do this would be to copy the beginning of the usage section in the release notes:

>Please refer to the ReadMe file for your OS.
>- macOS: see README_macOS.
>- Windows: see README_Windows
>- Linux: see README_Linux.

It would also be natural to include the complete list of packages and their versions in the release notes.

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : yes, I can definitely improve on the release notes. Currently CI for the version bumping + readme adjustment PR is running, so I will do the tag tomorrow. It would be great if we could discuss the release message tomorrow morning.

**Comment by Zimmi48** [comment]
With the change of default branch, it doesn't seem as useful as before to point to the releases section rather than the main repo page, so I'll revert this update to the previous version and merge.
